.drone.yml

```diff
@@ -0,0 +1,35 @@
+---
+kind: pipeline
+type: docker
+name: default
+
```

Comment:
lock to a platform like this:
https://github.com/rancher/dapper/blob/master/.drone.yml#L6-L8

Reply:
:heavy_check_mark: